schemas/dbt_yml_files.json

```diff
@@ -272,6 +272,18 @@
             },
             "timestamp": {
               "type": "string"
+            },
+            "window": {
+              "type": "object",
+              "properties": {
+                "count": {
+                  "type": "integer"
+                },
+                "period": {
+                  "type": "string",
+                  "enum": [ "day", "week", "month", "year", "all_time"]
```

Comment:
If you choose `all_time`, is it invalid to provide `count`? I assume so but that isn't reflected here

Reply:
Oh I didn't even think about that - yea `all_time` wouldn't work in window. The only unbounded cumulative metrics we offer are in secondary cacls. So removing all_time should fix this up!

Reply:
@callum-mcdata FYI i did steal this from the docs site :) 

will remove!

Reply:
Woops, that should be there! Let me remove from the docs